common/lib/common/kconfig.ts

```diff
@@ -389,7 +389,10 @@ const explorerCollectionUrl = (chainId: number, collAddress = ""): string => {
 
 // KREDEUM NFT URL
 const kredeumNftUrl = (chainId: number, nft: NftType): string =>
-  `./#/${getChainName(nft.chainId)}/${nft?.address}/${nft?.tokenID}`;
+  `#/${getChainName(nft.chainId)}/${nft?.address}/${nft?.tokenID}`;
```

Comment:
to check if direct NFT links still works with this

Reply:
made this to make it work in wordpress.
Checked in dapp that stills working.

are you talking about another direct NFT link ?

Reply:
made this to make it work in wordpress.
Checked in dapp that stills working.

are you talking about another direct NFT link ?